README.md

```diff
@@ -1,6 +1,16 @@
-# NeonRacer
+﻿# NeonRacer
 
-> ⚠️ Bitte nicht diesen Ordner in _IntelliJ_ öffnen! Der gesamte Code befindet sich im `src/` Ordner mitsamt einer Anleitung zum Konfigurieren des Projekts in _IntelliJ_.
+## Konfiguration
+
+Neon Racer wurde mit _IntelliJ Idea_ entwickelt. Zusätzlich sind noch _Git_ und ein _JDK_ erforderlich.
```

Comment:
Sollen wir schreiben "JDK" oder "JDK 1.8"

Reply:
> Wie machen wir das jetzt, dass das in allen Branches geupdated wird?

Das mache ich einfach in der Konsole mit `git merge`. Ich werde ein paar Dateien manuell verschieben müssen, aber das ist relativ wenig Arbeit.